ci-operator/config/openshift-kni/cluster-group-upgrades-operator/openshift-kni-cluster-group-upgrades-operator-main.yaml

```diff
@@ -33,6 +43,32 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws
+- as: ci-job
+  commands: make ci-job
+  container:
+    from: src
+- as: integration
+  steps:
+    cluster_profile: aws
+    env:
+      OO_CHANNEL: alpha
+      OO_INSTALL_NAMESPACE: openshift-cluster-group-upgrades
+      OO_PACKAGE: cluster-group-upgrades-operator
+      OO_TARGET_NAMESPACES: '!all'
+    test:
+    - as: integration
+      cli: latest
+      commands: |
+        KUBECONFIG="${SHARED_DIR}/kubeconfig"
+        export KUBECONFIG
+        cp $(echo "${KUBECONFIG}") /go/src/github.com/openshift-kni/cluster-group-upgrades-operator/kubeconfig
```

Comment:
Is having a local kubeconfig this required by the tests?

Reply:
I asked myself the same question here. I've tried it without this and the tests are failing. It seems to be a particularity of KUTTL. I've found another example here: https://github.com/openshift/release/blob/master/ci-operator/config/backube/volsync/backube-volsync-main.yaml#L116, the only other openshift/release example I could find that's using KUTTL in a similar manner.